tabbycat/draw/utils.py

```diff
@@ -22,3 +31,44 @@ def annotate_npullups(teams, until):
 
     for team in teams:
         team.npullups = npullups_by_team_id.get(team.id, 0)
+
+
+def send_mail_to_adjs(round):
+    tournament = round.tournament
+    draw = round.debate_set.all().prefetch_related('debateadjudicator_set')
```

Comment:
We probably want to prefetch `debateadjudicator_set__adjudicator`, since we need adjudicators' names?